performance-test/scripts/InjiWeb_Test_Script.jmx

```diff
@@ -0,0 +1,2119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
+      <boolProp name="TestPlan.functional_mode">false</boolProp>
+      <boolProp name="TestPlan.serialize_threadgroups">false</boolProp>
+      <elementProp name="TestPlan.user_defined_variables" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments"/>
+      </elementProp>
+      <stringProp name="TestPlan.user_define_classpath"></stringProp>
+    </TestPlan>
+    <hashTree>
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments">
+          <elementProp name="protocol" elementType="Argument">
+            <stringProp name="Argument.name">protocol</stringProp>
+            <stringProp name="Argument.value">https</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="serverIP" elementType="Argument">
+            <stringProp name="Argument.name">serverIP</stringProp>
+            <stringProp name="Argument.value">inji.cellbox1.mosip.net</stringProp>
```

Comment:
S02 T02 Issuer Id - issuer id to be configurable, it is is hardcoded to Esignet

Reply:
here also - > v1/mimoto/get-token/ESignet

Reply:
With respect to scenario 02 : S02 User choosing an issuer and landing on credential types screen (Execution)
I have added JSR223 pre-processor to handle 2 issuer , eSignet and Sunbird.

With respect to scenario 03: S03 User downloads MOSIP National ID credential (Execution)
v1/mimoto/get-token/ESignet
We can't modify eSignet, since we are performing only eSignet authentication. And Sunbird insurance is out of scope.

Reply:
v1/mimoto/get-token/ESignet - the end esignet is just a configuration. In another env you will see a different name.

Reply:
On S02 T02 Issuer Id, as per the request sheet it says only national id. why do you want to include sunbird ?
Just take the issuer id from the user defined variable

Reply:
@vishwa-vyom . Assigned userdefined variable name as issuerName, And provided value as ESignet.